src/app/story/story.component.html

```diff
@@ -5,12 +5,13 @@
   <div class="diagram" id="js-diagram">
     <ng-container *ngIf="story">
       <app-storyrow [node]="node" [backwardSiblings]="story.meta.story_related_nodes_backward[i]"
-          [forwardSiblings]="story.meta.story_related_nodes_forward[i]" [last]="last"
-          [active]="node === selectedNode"
-          *ngFor="let node of story.data; let i = index; let last = last">
+          [forwardSiblings]="story.meta.story_related_nodes_forward[i]" [first]="first" [last]="last"
+          [active]="node === selectedNode" [totalLeadTime]="story.meta.total_lead_time"
```

Comment:
Optional: Rather than 3 properties, do you think it makes sense to just pass in `story.meta` as a single property into `StoryRowComponent`?

Reply:
I think that would look nicer, but do you think it would affect readability much? It might be less confusing if we just get the data we want straight from the source at the beginning.

Reply:
I don't think it changes the readability. It's just less properties to manage, and if we need to use additional metadata to the story row component, we don't have to add another property.